### ISSUE TITLE
 Check failed: sss_[idx]->Get(key, &value) Failed to seek to the record for tensor

### ISSUE THREAD

**Issue by tengshaofeng** [tensorflow/tensorflow#11780]
hi, 
 I am so appreciated with your great job. 
here is my question:
when i finetune the resnet_v1_50 with pretrained model.   I have exclude the logists layer, when comes to 
the code 'saver.restore(sess, checkpoint_path)', it gives me the error like this:
![image](https://user-images.githubusercontent.com/9246739/28619251-144e03be-723a-11e7-8612-26c2cde56b01.png)
can you help me?

**Comment by aselle** [comment]
does it work if you don't exclude logits before loading the checkpoint? if so, load the checkpoint but ignore that logits layer. However, this is more of a stackoverflow question.
